src/Microsoft.IdentityModel.Protocol.Extensions/AudienceValidator.cs

```diff
@@ -31,19 +32,21 @@ internal class AudienceValidator
     {
         public static void Validate(IEnumerable<string> audiences, SecurityToken securityToken, TokenValidationParameters validationParameters)
         {
+            IdentityModelEventSource.Logger.WriteInformation("AudienceValidator.Validate: validating audience in the jwt token");
+
             if (audiences == null)
             {
-                throw new ArgumentNullException("audiences");
+                LogHelper.LogError(string.Format(CultureInfo.InvariantCulture, ErrorMessages.IDX10000, "AudienceValidator.Validate: audiences"), typeof(ArgumentNullException));
```

Comment:
I don't think we need to include 'AudienceValidator: Validate' that will be in the stack trace. It also gets stale if we ever change the method name or class which could happen if the code is reused or forked.